README.md

```diff
@@ -4,301 +4,41 @@
 [![Go Reference](https://pkg.go.dev/badge/github.com/wavefronthq/wavefront-sdk-go.svg)](https://pkg.go.dev/github.com/wavefronthq/wavefront-sdk-go)
 [![Go Report Card](https://goreportcard.com/badge/github.com/wavefronthq/wavefront-sdk-go)](https://goreportcard.com/report/github.com/wavefronthq/wavefront-sdk-go)
 
-## Table of Content
-* [Prerequisites](#Prerequisites)
-* [Set Up a Sender](#set-up-a-sender)
-* [Send Data to Wavefront](#send-data-to-wavefront)
-* [Close the Sender](#close-the-sender)
+## Table of Contents
 * [License](#License)
 * [How to Get Support and Contribute](#how-to-get-support-and-contribute)
-* [How to Release the SDK](#how-to-release-the-sdk)
 
 # Welcome to the Wavefront Go SDK
 
-Wavefront by VMware Go SDK lets you send raw data from your Go application to Wavefront using a `Sender` interface. The data is then stored as metrics, histograms, and trace data. This SDK is also called the Wavefront Sender SDK for Go. 
+Wavefront by VMware Go SDK lets you send raw data from your Go application to Wavefront using a `Sender` interface.
+The data is then stored as metrics, histograms, and trace data. This SDK is also called the Wavefront Sender SDK for Go. 
 
-Although this library is mostly used by the other Wavefront Go SDKs to send data to Wavefront, you can also use this SDK directly. For example, you can send data directly from a data store or CSV file to Wavefront.
+Although this library is mostly used by the other Wavefront Go SDKs to send data to Wavefront, 
+you can also use this SDK directly. For example, you can send data directly from a data store or CSV file to Wavefront.
 
-**Before you start implementing, let us make sure you are using the correct SDK!**
+For detailed documentation of SDK types and functions, see [pkg.go.dev](https://pkg.go.dev/github.com/wavefronthq/wavefront-sdk-go/senders)
```

Comment:
To lean more on how to send data, the SDK types, and functions, see [pkg.go.dev documentation](https://pkg.go.dev/github.com/wavefronthq/wavefront-sdk-go/senders)